core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -234,20 +235,14 @@ public Set<String> getScanServers() {
 
   @Override
   public List<String> getTabletServers() {
-    ZooCache cache = context.getZooCache();
-    String path = context.getZooKeeperRoot() + Constants.ZTSERVERS;
+    Set<ServiceLockPath> paths =
+        context.getServerPaths().getTabletServer(Optional.empty(), Optional.empty());
     List<String> results = new ArrayList<>();
-    for (String candidate : cache.getChildren(path)) {
-      var children = cache.getChildren(path + "/" + candidate);
-      if (children != null && !children.isEmpty()) {
-        var copy = new ArrayList<>(children);
-        Collections.sort(copy);
-        var data = cache.get(path + "/" + candidate + "/" + copy.get(0));
-        if (data != null && !"manager".equals(new String(data, UTF_8))) {
-          results.add(candidate);
-        }
+    paths.forEach(p -> {
+      if (!p.getServer().equals("manager")) {
```

Comment:
What is going on with this check for `manager`?  Is this still needed?

Reply:
I don't know. I just tried to preserve it.

Reply:
Ok I will do some digging and see if I can figure anything out.